README.md

```diff
@@ -29,6 +31,25 @@ pod 'Valet'
 
 Or manually checkout the submodule with `git submodule add git@github.com:Square/Valet.git`, drag Valet.xcodeproj to your project, and add Valet as a build dependency.
 
+### Carthage
+
+[Carthage](https://github.com/Carthage/Carthage) is a decentralized dependency manager that builds your dependencies and provides you with binary frameworks.
+
+You can install Carthage with [Homebrew](http://brew.sh/) using the following command:
+
+```bash
+$ brew update
+$ brew install carthage
+```
+
+To integrate Valet into your Xcode project using Carthage, specify it in your `Cartfile`:
+
+```ogdl
+github "Square/Valet" ~> 2.0.3
```

Comment:
Do Carthage users need to be this specific? Could they say `~> 2.0`? I'd prefer not to prescribe a bug fix version in the README if that's not necessary.